zebrad/tests/common/configs/v1.7.0.toml

```diff
@@ -61,6 +61,16 @@ network = "Testnet"
 peerset_initial_target_size = 25
 
 [network.testnet_parameters]
+activation_heights = [
+    [0, "Genesis"],
+    [1, "BeforeOverwinter"],
```

Comment:
Genesis is certainly required, `BeforeOverwinter` is probably the same as `BASE_SPROUT` in zcashd so it probably also has to always be 1.